var/spack/repos/builtin/packages/osu-micro-benchmarks/package.py

```diff
@@ -20,7 +20,7 @@ class OsuMicroBenchmarks(AutotoolsPackage, CudaPackage):
 
     maintainers = ["natshineman", "harisubramoni", "MatthewLieber"]
 
-    version("6.1", sha256="7f3be7526411449a765797ee1db7cee14a79a3d536e01063703457469fb9d0a3")
+    version("6.1", sha256="ecccedc868264f75db4d9529af79005419a2775113c7fae8f4e4a8434362e4a7")
```

Comment:
Did anything change in the package? Do we have to expect changes without version bumping for `osu-micro-benchmarks`?

Reply:
We identified an error in the CHANGELOG that indicated the wrong flag for a newly added feature. This change was purely documentation. 

In the future this should not be an issue and we do not expect further changes to the package after release.